packages/react-icons-mdl2-branded/LICENSE

```diff
@@ -0,0 +1,15 @@
+@fluentui/react-icons-mdl2-branded
+
+Copyright (c) Microsoft Corporation
+
+All rights reserved.
+
+MIT License
```

Comment:
Is this the right license for these icons?

Reply:
Changed to Microsoft Fabric Assets License: https://static2.sharepointonline.com/files/fabric/assets/microsoft_fabric_assets_license_agreement_nov_2019.pdf